galaxy_ng/app/management/commands/dump_auth_config.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from dynaconf.utils import upperfy
+
+
+class Command(BaseCommand):
+    KEYCLOAK_AUTH_KEYS = [
+        "SOCIAL_AUTH_KEYCLOAK_ACCESS_TOKEN_URL",
+        "SOCIAL_AUTH_KEYCLOAK_AUTHORIZATION_URL",
+        "SOCIAL_AUTH_KEYCLOAK_KEY",
+        "SOCIAL_AUTH_KEYCLOAK_PUBLIC_KEY",
+        "SOCIAL_AUTH_KEYCLOAK_SECRET",
+        "KEYCLOAK_PROTOCOL",
+        "KEYCLOAK_HOST",
+        "KEYCLOAK_PORT",
+        "KEYCLOAK_REALM",
+    ]
+
+    LDAP_AUTH_KEYS = [
+        "AUTH_LDAP_SERVER_URI",
+        "AUTH_LDAP_BIND_DN",
+        "AUTH_LDAP_BIND_PASSWORD",
+        "AUTH_LDAP_USER_SEARCH_BASE_DN",
+        "AUTH_LDAP_USER_SEARCH_SCOPE",
+        "AUTH_LDAP_USER_SEARCH_FILTER",
+        "AUTH_LDAP_GROUP_SEARCH_BASE_DN",
+        "AUTH_LDAP_GROUP_SEARCH_SCOPE",
+        "AUTH_LDAP_GROUP_SEARCH_FILTER",
+        "AUTH_LDAP_USER_ATTR_MAP",
+    ]
+
+    help = "Dump auth config data from database to a JSON file"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "output_file",
+            nargs="?",
+            type=str,
+            default="/auth_config.json",
+            help="Output JSON file path",
+        )
+
+    def format_config_data(self, type, keys, prefix):
+        config = {
+            "type": type,
+            "configuration": {}
+        }
+        for key in keys:
+            k = key
+            if prefix in key:
+                k = key[len(prefix):]
+            v = settings.get(upperfy(key), default=None)
```

Comment:
is this call to `upperfy` needed?
dynaconf uses this when setting new variables, but for acessing variables it is case insensitive .get('FOO') .get('foo') and even .get('FoO') would find the same value.

Reply:
Sounds like it's not needed then, I didn't realize it was case insensitive at that point.  Thanks!